backend/app/models/user.rb

```diff
@@ -19,11 +19,11 @@ def self.find_by_credentials(username, password)
     user = User.find_by(username: username)
     return nil unless user
 
-    user.password?(password) ? user : nil
+    user.is_password?(password) ? user : nil
   end
 
-  def password?(password)
-    BCrypt::Password.new(password_digest).password?(password)
+  def is_password?(password)
```

Comment:
I don't think this is needed. Not sure what the editor was telling you but I think rubocop convention is to leave this as `password?`